src/windows/list.cpp

```diff
@@ -579,12 +584,13 @@ std::vector<DeviceDescriptor> ListStorageDevices() {
     device.isSCSI = IsSCSIDevice(enumeratorName);
     device.isUSB = IsUSBDevice(enumeratorName);
     device.isCard = device.enumerator == "SD";
-    // device.isSystem = IsSystemDevice(hDeviceInfo, deviceInfoData);
-    device.isSystem = device.isSCSI && !device.isRemovable;
+    device.isSystem = !device.isRemovable &&
+      (device.enumerator == "SCSI" || device.enumerator == "IDE");
```

Comment:
Are the strings here the same as those listed at https://github.com/resin-io-modules/drivelist/blob/master/src/windows/list.cpp#L289 ? As that doesn't mention "IDE". Or are these something else?
Are there any enumerators other than "SCSI" and "IDE" that we should always consider as being a "system drive"?

As an aside, should we have some kind of "definition" of what exactly a "system drive" is (up until now, it has simply indicated "non removable"), with our code then making best-effort attempts to match up to that definition? IYSWIM.